app/src/main/res/values/style.xml

```diff
@@ -24,4 +24,16 @@
         <item name="android:navigationBarColor">@color/evenDarker</item>
         <item name="android:windowBackground">@color/dark</item>
     </style>
+
+    <style name="TabsTrayThumbnailStyle" parent="@style/Widget.MaterialComponents.CardView">
```

Comment:
Let's remove this since the MaterialCardView doesn't seem to work well for our use case - sorry about that!